### DISCUSSION TITLE
change icon from menu

### DISCUSSION THREAD

**Discussion by jorgeucano** [sibiraj-s/ngx-editor#454] [Q&A] [answered]
Is it possible to change the menu icons? or you just need to create a custom menu? the problem is rewriting all the rules to support the functionality...
So, if I have the possibility to change only the icons and add some styles, it would help me a lot so as not to rewrite everything that already exists.

thanks

**Comment by sibiraj-s** [comment]
Yeah. Currently there are no such options. Might be a good addition though. Let me know if you have any suggestions on implementation.

**Reply by jorgeucano** [reply]
I created this pr with the implementation for support any icon (tag i, img, svg, any other) https://github.com/sibiraj-s/ngx-editor/pull/455

**Comment by sibiraj-s** [comment, marked as answer]
Implemented in [v15.3.0](https://github.com/sibiraj-s/ngx-editor/releases/tag/v15.3.0)

[a comment by an automated account is not shown]
